### DISCUSSION TITLE
Welcome to deepflame-dev Discussions!

### DISCUSSION THREAD

**Discussion by zhixchen** [deepmodeling/deepflame-dev#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our DeepFlame community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shock1992** [comment]
Dear Prof. Chen, would you please add references' information to the FGM model, such that it will be much easier to be understood.

**Reply by zhixchen** [reply]
Thank you for raising this point. We have added a README in this PR #243.

**Comment by MacCary** [comment]
Hello, I'm sorry to bother you. I have some questions to ask you. I really hope you can answer them. I had some problems installing deepflame. The first question is: there is no envt/deepflame/lib/include/etc this folder. The second is that there is no error during the compilation process, but it often appears that could not find XXX file or dictonry, one of the files that could not be found is the fmt/format.h header file, and others (I do not know whether these header files are in the folder I just said). As a result, four of deepflame's built-in solvers did not work properly. I'm sorry to send you a private message in my busy schedule. I hope you can answer it. Thank you very much.

**Reply by xiao312** [reply]
Hi!

Thank you for contacting us about your issue.

To better assist you with this issue, could you please provide additional information on every step you took during the installation process? Having this information will enable us to investigate the issue further and work towards finding a resolution.

Best regards!

**Comment by SnKzm** [comment]
Hello,

I recently modified my soot solver to have consistent syntax as DeepFlame (let's say Cantera) to use DeepFlame instead of the original OpenFOAM solvers. However, during the implementation of various source terms related to soot formation into the energy equation of dfLowMachFoam, I noticed that the Qdot source term (which adds heat from reactions) doesn't appear in EEqn!!! So, I wonder if anyone can help me understand how DeepFlame considers the heat of reactions when solving the energy equation.

I hasten to add that I reviewed the DeepFlame publication and saw that there is no source term due to reactions there as well.

**Reply by zhixchen** [reply]
Hi, 

We use total enthalpy (chemical+sensible) in dfLowMachFoam. It is a conserved quantity during constant pressure combustion. Thus, there is no source term.

**Comment by mactone** [comment]
Is the deepflame restricted to openfoam7+ubuntu20.04?
Could I use other version combinations, such as openfoam 2206+ubuntu 24.04 ?

**Reply by SnKzm** [reply]
I used it on different Ubuntu and CentOS operating systems, and it works fine. You can probably run it on Ubuntu 24.04 as well without any issues (although I am not certain!).

Regarding OpenFOAM 2206, please note that there are significant differences between the OpenFOAM Foundation version and the version you mentioned. You won’t be able to use the main version of DeepFlame in conjunction with OpenFOAM 2206. However, if you check GitHub, there is a version (branch 2212) that is likely compatible with OpenFOAM 2206.
